modules/virt-importing-vm-datavolume.adoc

```diff
@@ -106,6 +106,7 @@ status: {}
 <1> The `HTTP` source of the image you want to import.
 <2> The `secretRef` parameter is optional.
 <3> The `certConfigMap` is required for communicating with servers that use self-signed certificates or certificates not signed by the system CA bundle. The referenced ConfigMap must be in the same namespace as the DataVolume.
+<4> Use `type: datavolume` or `type: ""` (default for dataVolume). If you use other types such as `persistentVolumeClaim`, a warning will be triggered and the virtual machine fails to start.
```

Comment:
s/Use/Specify

Is there any impact to specifying `datavolume`? If so, please explain. Also, rewrite to remove the parentheses.

> Avoid using parentheses in running text because text in parentheses is decreased in importance.

I'd replace the second sentence because it has some grammar and style issues.
> If you specify other types, such as `persistentVolumeClaim`, a warning is displayed, and the virtual machine does not start.

Reply:
Changed to: 
Specify `type: dataVolume` or `type: ""`. If you specify any other value for `type`, such as `persistentVolumeClaim`, a warning is displayed and the virtual machine does not start.

Corrected to "dataVolume" for correct YAML syntax.